src/services/config-utility.js

```diff
@@ -0,0 +1,9 @@
+const availableConfigs = ['edge'];
+function getFromEnvironment(config) {
+  const configEnv = `TWILIO_${config.toUpperCase()}`;
+  if (process.env[configEnv]) {
+    return `${process.env[configEnv]}`;
```

Comment:
Why not process.env[configEnv] ?

Reply:
Ah, I see you updated in next revision. Let's add UTs to catch such scenarios? 
Also- Doesn't the coverage results show up 0% for this file (and drop coverage % for the package?)

Reply:
As discussed, This code will be covered as part of config commands UT's itself, why UT's didn't fail is because `process.env[configEnv]` and `${process.env[configEnv]}` will result in the same value.